src/base/bittorrent/session.cpp

```diff
@@ -3929,6 +3929,37 @@ quint64 Session::getAlltimeUL() const
     return m_statistics->getAlltimeUL();
 }
 
+QVariantMap Session::getStats() const
```

Comment:
I felt `getStats()` is a bit too generic, it should return all stats but it did not.
Maybe `getOrganizedStats()`?

Reply:
At least, it's a really bad idea to return some kind of serialized data here. And, therefore, this method is completely useless, because it contains only trivial logic. It does not add any convenience, and does not correct any errors. Why not just do the same things where they are needed?

Reply:
Will rename to `getOrganizedStats()`